buildSrc/src/main/java/Dependencies.kt

```diff
@@ -50,6 +50,9 @@ object Versions {
     const val google_ads_id_version = "16.0.0"
 
     const val google_play_store_version = "1.8.0"
+    const val google_services_version = "4.3.4"
+    const val firebase_crashlytics = "17.2.2"
```

Comment:
This is may be a problem in the future if we try to upgrade a version here or AC since we know that individual Firebase SDKs need to have compatible versions.

Probably the same comment for `firebase_crashlytics_gradle` as well.

Reply:
I agree.  I was a bit worry about this as well.  But I couldn't find a way that I can add this only on the A-C side.